lighthouse-cli/bin.ts

```diff
@@ -278,7 +282,8 @@ function saveResults(results: Results,
 
 function runLighthouse(url: string,
                        flags: {port: number, skipAutolaunch: boolean, selectChrome: boolean, output: any,
-                         outputPath: string, interactive: boolean, saveArtifacts: boolean, saveAssets: boolean},
+                         outputPath: string, interactive: boolean, saveArtifacts: boolean, saveAssets: boolean
+                         timeout: number},
```

Comment:
is this just for documentation? I wouldn't think TS would care since we don't use it here